typeMap.xml

```diff
@@ -965,33 +965,33 @@
     </itemTypes>
     <vars>
       <var name="abstract" type="standard" description="abstract of the item (e.g. the abstract of a journal article)" />
-      <var name="annote" type="standard" description="reader's notes about the item content" />
+      <var name="annote" type="standard" description="reader’s notes about the item content" />
       <var name="archive" type="standard" description="archive storing the item" />
       <var name="archive_location" type="standard" description="storage location within an archive (e.g. a box and folder number)" />
       <var name="archive-place" type="standard" description="geographic location of the archive" />
-      <var name="authority" type="standard" description="issuing or judicial authority (e.g. &quot;USPTO&quot; for a patent, &quot;Fairfax Circuit Court&quot; for a legal case)" />
+      <var name="authority" type="standard" description="issuing or judicial authority (e.g. “USPTO” for a patent, “Fairfax Circuit Court” for a legal case)" />
```

Comment:
This is fetched directly from http://citationstyles.org/downloads/specification.html which now uses Sphinx for rendering and that [defaults to converting quotes](http://www.sphinx-doc.org/en/stable/config.html#confval-html_use_smartypants). I don't think this will be an issue.